### PR TITLE
add ERC721PoolInfoUtils tests

### PR DESCRIPTION
**Pull request by MikeHathaway**
## Description

- Expand code coverage by adding tests for 721 pool flow of  `testPoolInfoUtilsMulticallPoolBalanceDetails`.
- Cover `currentBurnEpoch` and `burnInfo`.

### CODE REVIEW

File: tests/forge/unit/ERC721Pool/ERC721PoolInfoUtils.t.sol [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        subsetTokenIds[0] = 1;
+        subsetTokenIds[1] = 3;
+        subsetTokenIds[2] = 5;
+        subsetTokenIds[3] = 51;
+        subsetTokenIds[4] = 53;
+        subsetTokenIds[5] = 73;
+        return _deploySubsetPool(subsetTokenIds);
+    }
+
+    function testPoolInfoUtilsInvariantsFuzzed(uint256 depositIndex_, uint256 price_) external {
```
Comment (prat-gpt) [inline tests/forge/unit/ERC721Pool/ERC721PoolInfoUtils.t.sol:85]: I think this test is redundant and has already been written here: https://github.com/ajna-finance/contracts/blob/develop/tests/forge/unit/ERC20Pool/ERC20PoolInfoUtils.t.sol#L89-L95

**Review by prat-gpt — APPROVED** [review]
LGTM. Added a comment

File: tests/forge/unit/ERC721Pool/ERC721PoolInfoUtils.t.sol (hunk 1 above)
Comment (EdNoepel) [reply tests/forge/unit/ERC721Pool/ERC721PoolInfoUtils.t.sol:85]: But this instance covers ERC721 pools.

**Review by EdNoepel — APPROVED** [review]
LGTM

File: tests/forge/unit/ERC721Pool/ERC721PoolInfoUtils.t.sol (hunk 1 above)
Comment (prat-gpt) [reply tests/forge/unit/ERC721Pool/ERC721PoolInfoUtils.t.sol:85]: This test checks deposit index and price of the bucket which is independent of the type of pool. So I think as it is already checked in ERC20 pool, it makes this test redundant.

**Review by ith-harvey — APPROVED** [review]
LGTM

**Pull request merged**